promscale/visualize-data/jaeger.md

```diff
@@ -0,0 +1,38 @@
+# Connect Promscale and Jaeger Query
+This section shows you how to connect Promscale with
+[Jaeger][grafana-jaeger-datasource].
+
+Before you begin, you need the IP address of your Promscale instance. You can
```

Comment:
These are very specific instructions to Docker. It will depend on where folks run Promscale. If they don't run it in a container then these instructions would not work.